### PR TITLE
Course discovery static files

### PR DESCRIPTION
**Pull request by cpennington**
@e0d, @feanil: Can one of your review?
@clintonb: FYI

This relies on https://github.com/edx/course-discovery/pull/7

### CODE REVIEW

File: playbooks/roles/course_discovery/templates/edx/app/course_discovery/devstack.sh.j2 [hunk 1]
```diff
@@ -15,6 +10,12 @@ case $COMMAND in
         {% set course_discovery_venv_bin = course_discovery_home + "/venvs/" + course_discovery_service_name + "/bin" %}
         {% set executable = course_discovery_venv_bin + '/gunicorn' %}
 
+        cd /edx/app/edx_ansible/edx_ansible/docker/plays
+        sudo ansible-playbook course_discovery.yml -c local -i '127.0.0.1,' \
+            -t 'install:app-requirements,assets:gather,migrate:devstack,manage:devstack' \
```
Comment (feanil) [inline playbooks/roles/course_discovery/templates/edx/app/course_discovery/devstack.sh.j2:15]: you've got 'manage:devstack' in here twice

File: playbooks/roles/ansible-role-django-ida/templates/templates/edx/app/ROLE_NAME/devstack.sh.j2 [hunk 2]
```diff
@@ -15,6 +10,12 @@ case $COMMAND in
         {{ '{%' }} set {{ role_name }}_venv_bin = {{ role_name }}_home + "/venvs/" + {{ role_name }}_service_name + "/bin" %}
         {{ '{%' }} set executable = {{ role_name }}_venv_bin + '/gunicorn' %}
 
+        cd /edx/app/edx_ansible/edx_ansible/docker/plays
+        sudo ansible-playbook {{ role_name }}.yml -c local -i '127.0.0.1,' \
+            -t 'install:app-requirements,assets:gather,migrate:devstack,manage:devstack' \
```
Comment (feanil) [inline playbooks/roles/ansible-role-django-ida/templates/templates/edx/app/ROLE_NAME/devstack.sh.j2:15]: `migrate:devstack` is in here twice.

File: docker/plays/ansible.cfg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[defaults]
```
Comment (feanil) [inline docker/plays/ansible.cfg:1]: What was the reason for moving this into plays?  Does nothing run ansible from the build directory now?

File: playbooks/roles/ansible-role-django-ida/templates/templates/edx/app/ROLE_NAME/devstack.sh.j2 (hunk 2 above)
Comment (cpennington) [inline playbooks/roles/ansible-role-django-ida/templates/templates/edx/app/ROLE_NAME/devstack.sh.j2:15]: One is `migrate`, the other is `manage`.

File: playbooks/roles/course_discovery/templates/edx/app/course_discovery/devstack.sh.j2 (hunk 1 above)
Comment (cpennington) [inline playbooks/roles/course_discovery/templates/edx/app/course_discovery/devstack.sh.j2:15]: one is `migrate`, the other is `manage`.

File: docker/plays/ansible.cfg (hunk 3 above)
Comment (cpennington) [inline docker/plays/ansible.cfg:1]: As far as I could tell, things are only run from `plays` (that said, `course_discovery` is the only Dockerfile at the moment). In general, `docker/build` is about docker, and `docker/plays` is about ansible, so moving the `ansible.cfg` to the plays directory made sense to me.

Comment (feanil) [inline docker/plays/ansible.cfg:1]: Cool, makes sense.

**Comment by feanil** [comment]
:+1:

**Pull request merged**